### PR TITLE
chores: add more docs and fix some bugs.

### LINKED ISSUE #886: [Doc] We need more detailed documentation of how label data works.
As more and more label data comes into OpenDigger, we need a more detailed documentation of how label data works and how to add new label data into the project like how to choose the granularity of label.

### PR DESCRIPTION
**Pull request by xgdyp**
Hi, in this pr, I update data_utils to support new label type such as `Tech-0`.
I also changed head_report direction from `/cooperation` to `cooperations/big_data_open_source_heat_report/`
Also, I find we need more docs on labeled_data so I write some docs

close #886

### CODE REVIEW

File: labeled_data/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+In this folder, we collect different categories of labeled data. We can using these data by adding `labelIntersect` parameter in metrics like:
+``` 
+openDigger.index.activity.getRepoActivity({labelIntersect: ['Company', ':regions/China'], startYear, startMonth, endYear, endMonth, groupBy: 'Company', groupTimeRange: 'year', limit: -1, order: 'DESC' })
+```
+We have 6 categories of labeled data.
+
+#### Data Description
```
Comment (frank-zsy) [inline labeled_data/README.md:7]: Why use a 4th level title when there are no other levels?

File: labeled_data/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- projects
+It contains repos/orgs used around a certain project.
+
+- regions
+It contains repos/orgs/users that (created) from different regions. 
+
+- technology
+It contains repos in different technical field.
+
```
Comment (frank-zsy) [inline labeled_data/README.md:25]: Why there are 3 blank lines?

File: labeled_data/README.md (hunk 1 above)
Comment (xgdyp) [reply labeled_data/README.md:7]: I'll fix this.

**Comment by frank-zsy** [comment]
Thanks, I will add more docs about label calculation later.

/approve

**Pull request merged**